index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for jasmine-data-provider 2.2.0
```

Comment:
Is the version here necessary? And should I increase this version with every release?

Reply:
Maybe reference is missing. https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html